theory.md

```diff
@@ -7,6 +7,8 @@ a. Single regex that matches either of these:
     antelope rocks out
     
     antelopes rock out
+    
+    regex = r'[a]s?\w+'
```

Comment:
In this exercise we want to target antelope or antelopes. This regex will target anything that's starting on letter `A`.

You could do something like `/antelopes?/` which will target antelope or antelopes.